native/src/main/java/io/ballerina/stdlib/oauth2/OAuth2Client.java

```diff
@@ -104,20 +103,18 @@ public static Object doHttpRequest(BString url, BMap<BString, Object> clientConf
             } catch (Exception e) {
                 return createError("Failed to init SSL context. " + e.getMessage());
             }
-        }
+        }        
```

Comment:
Remove additional spaces. You can configure your IDE to remove the trailing spaces when saving a file.

```suggestion
        }
```